python/lsst/sdm/schemas/efd_latiss.yaml

```diff
@@ -0,0 +1,2764 @@
+  - name: "at_salindex205_wind_speed_speedstddev_2_min"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_speedstddev_2_min"
+    datatype: float
+    nullable: true
+    description: Minimum standard deviation of wind speed (x, y, z) estimated from quartiles in AuxTel-GillLabJack01 along axis 2 (salIndex 205).
+  - name: "at_salindex205_wind_speed_speedstddev_2_max"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_speedstddev_2_max"
+    datatype: float
+    nullable: true
+    description: Maximum standard deviation of wind speed (x, y, z) estimated from quartiles in AuxTel-GillLabJack01 along axis 2 (salIndex 205).
+  - name: "at_salindex205_wind_speed_magnitude_mean"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_magnitude_mean"
+    datatype: float
+    nullable: true
+    description: Mean median wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_magnitude_stddev"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_magnitude_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the median wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_magnitude_min"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_magnitude_min"
+    datatype: float
+    nullable: true
+    description: Minimum median wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_magnitude_max"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_magnitude_max"
+    datatype: float
+    nullable: true
+    description: Maximum median wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_maxmagnitude_mean"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_maxmagnitude_mean"
+    datatype: float
+    nullable: true
+    description: Mean maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_maxmagnitude_stddev"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_maxmagnitude_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_maxmagnitude_min"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_maxmagnitude_min"
+    datatype: float
+    nullable: true
+    description: Minimum maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_maxmagnitude_max"
+    "@id": "#visit1_efd.at_salindex205_wind_speed_maxmagnitude_max"
+    datatype: float
+    nullable: true
+    description: Maximum maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex201_airflow_speed_mean"
+    "@id": "#visit1_efd.at_salindex201_airflow_speed_mean"
+    datatype: float
+    nullable: true
+    description: Mean median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
```

Comment:
More "UNKNOWN" and in following columns.

Reply:
Fixed.